queries/goctl/indents.scm

```diff
@@ -0,0 +1,33 @@
+[
+  (importGroup)
+  (typeGroupSpec)
+  (structType)
+  (infoStatement)
+  (serviceServerSpec)
+  (serviceSpec)
+] @indent.begin
+
+(importGroup
+  ")" @indent.branch)
+
+(structType
+  "}" @indent.branch)
+
+(typeGroupSpec
+  ")" @indent.branch)
+
+(infoStatement
+  ")" @indent.branch)
+
+(serviceServerSpec
+  ")" @indent.branch)
+
+(serviceSpec
+  "}" @indent.branch)
+
+[
+  "}"
+  ")"
+] @indent.end
```

Comment:
Why can't this be reduced to `[ ")" "}" ] @indent.branch @indent.end`

Reply:
> Why can't this be reduced to `[ ")" "}" ] @indent.branch @indent.end`

I am not particularly clear here, I refer to several other documents to write, such as `go`.
Do you mean change the last piece to `[ ")" "}" ] @indent.branch @indent.end`?

Reply:
Moreso a suggestion that I thought of, seeing how the `@indent.branch` blocks above consist of all the nodes in `@indent.begin`